docs/source/managed-federation/deployment.mdx

```diff
@@ -212,38 +192,38 @@ extend type Product @key(fields: "upc") {
 
 If the Apollo Studio API experiences an outage, changes to your configuration will halt until the outage resolves, but your most recently published configuration for each variant is still accessible via GCS.
 
-## The `service:push` lifecycle
+## The `subgraph publish` lifecycle
 
-Whenever you call `apollo service:push` for an implementing service, it both updates that service's registered schema and updates the gateway's managed configuration. Because the graph is dynamically changing and multiple services might be updated simultaneously, it's possible for changes to cause composition errors, even if `apollo service:check` was successful. For this reason, updating a federated service will re-trigger composition in the cloud, ensuring that the federated services still compose to form a complete graph before provisioning the configuration. The workflow behind the scenes can be summed up as follows:
+Whenever you call `rover subgraph publish` for a particular subgraph, it both updates that subgraph's registered schema and updates the gateway's managed configuration. Because the graph is dynamically changing and multiple subgraphs might be updated simultaneously, it's possible for changes to cause composition errors, even if `rover subgraph check` was successful. For this reason, updating a subgraph re-triggers composition in the cloud, ensuring that all subgraphs still compose to form a complete supergraph before provisioning the configuration. The workflow behind the scenes can be summed up as follows:
 
-1. The partial schema is uploaded to a secure location and indexed.
-2. The federated service is updated in the registry to point to the partial schema.
-3. All federated services are composed in the cloud to produce a new complete schema.
+1. The subgraph schema is uploaded to a secure location and indexed.
+2. The subgraph is updated in the registry to point to the new schema.
+3. All subgraphs are composed in the cloud to produce a new supergraph schema.
 4. If composition fails, the command exits and emits errors.
-5. If composition succeeds, the top-level configuration file is updated in-place to point to the updated set of federated services.
+5. If composition succeeds, the top-level configuration file is updated in-place to point to the updated set of subgraphs.
 
 On the other side of the equation sits the gateway. The gateway is constantly listening for changes to the top-level configuration file. The location of the configuration file is guarded by using a hash of the API key, provisioned ahead of time so as not to affect reliability. The lifecycle of dynamic configuration updates is as follows:
 
 1. The gateway listens for updates to its configuration.
-2. On update, the gateway downloads the configuration for each implementing service in parallel.
+2. On update, the gateway downloads the configuration for each subgraph in parallel.
```

Comment:
This is not quite right in the new model - the gateway will pull down the supergraph artifact, not the individual subgraphs.

Reply:
Ah yeah, and number 3 is no longer true because the gateway doesn't compose anymore, correct?